src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,110 @@
+.main {
+  padding-top: 50px;
+}
+
+.product {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  &__card {
+    box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`